cmd/containerd-shim-runhcs-v1/service_internal.go

```diff
@@ -206,7 +206,6 @@ func (s *service) startInternal(ctx context.Context, req *task.StartRequest) (*t
 
 func (s *service) deleteInternal(ctx context.Context, req *task.DeleteRequest) (*task.DeleteResponse, error) {
 	// TODO: JTERRY75 we need to send this to the POD for isSandbox
-
```

Comment:
I think the TODO is intended to apply to the whole function, and removing the blank line makes that less clear.

Reply:
I don't think this was intentional, I had a whole bunch of debug logs to try and figure out why stopping a container was stuck after restart. Changing back